.github/workflows/ci.yml

```
@@ -0,0 +1,22 @@
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.23.0"
+
+      - name: Force Failure
+        run: (exit 1)
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)